src/types.ts

```diff
@@ -154,3 +154,4 @@ export type Voucher = {
   serial: string;
   denomination: number;
 };
+export type TimeoutMS = number;
```

Comment:
Hmm i think we can move this back into helper.ts since it's not shared elsewhere